src/main/java/org/team1540/robot2018/OI.java

```diff
@@ -61,9 +77,47 @@ private static double scale(double input, double pow) {
   public static final int RIGHT_X = 4;
   public static final int RIGHT_Y = 5;
 
-  // TODO: Remove unused button
-  // wrist to forward 45 degrees
-  // static Button copilotX = new JoystickButton(copilot, X);
+  public OI() {
+    // INTAKE
+    OI.intakeSequenceButton.whenPressed(new IntakeSequence());
+    OI.ejectButton.whenPressed(new JoystickEject());
+
+    OI.stopIntakeButton.whenPressed(new SimpleCommand("Stop intake", intake::holdCube, intake,
+        arms));
+
+    // ARMS
+    OI.intakeSequenceButton.whileHeld(new JoystickArms());
+    // OI.intakeSequenceButton.whileHeld(new SimpleCommand("Intake Arm Open", () -> arms.set
```

Comment:
Just remove it if we aren't going to use

Reply:
I left this as Alexander might want to switch back to the old method of controlling the intake arms